#! /bin/sh /usr/share/dpatch/dpatch-run
## 03jhexen.ded-debian-hook.dpatch by  <jamie_jones_au@yahoo.com.au>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This adds the jhexen-debian.ded that is responsible for ensuring
## DP: the jpacks are loaded, and that they are disabled when they
## DP: conflict with any mods or pwads.

@DPATCH@
diff -urNad deng-1.8.9+1.9.0beta2/Defs/jHexen/jHexen.ded /tmp/dpep.dLlFIn/deng-1.8.9+1.9.0beta2/Defs/jHexen/jHexen.ded
--- deng-1.8.9+1.9.0beta2/Defs/jHexen/jHexen.ded	2004-01-20 06:27:04.000000000 +1100
+++ /tmp/dpep.dLlFIn/deng-1.8.9+1.9.0beta2/Defs/jHexen/jHexen.ded	2005-08-14 02:21:27.262067568 +1000
@@ -24,6 +24,7 @@
 Include "Text.ded";
 Include "Values.ded";
 Include "Models.ded";
+Include "jhexen-debian.ded";
 Include "User.ded";
 
 IncludeIf Not -noauto "Auto\*.ded";
